tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -66,9 +66,11 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 			When("a nodejs component is created", func() {
 
 				BeforeEach(func() {
+					helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
```

Comment:
Not sure if this will have any role here. IIRC I had discussion with @dharmit around this and `operatorhub` tests are designed for testing functionality related to operators. We are not worried about the source code of the component. @dharmit Correct me if I am wrong.

Reply:
I would prefer to have these files that can help for the visibility of the logs: when running `odo push` withtout any source files, you get errors on the container logs because the build of the sources fails.

Reply:
@prietyc123 the reason given by @feloy makes sense.